.circleci/config.yml

```
@@ -38,7 +39,21 @@ jobs:
           command: |
             echo "$GCLOUD_SERVICE_KEY" > credentials.json
             echo "export GOOGLE_APPLICATION_CREDENTIALS=\"$(realpath credentials.json)\"" >> $BASH_ENV
+            google-cloud-sdk/bin/gcloud auth activate-service-account --key-file=credentials.json
+      - run:
+          name: "Set up k8s cluster"
```

```suggestion
          name: "Set up GKE cluster credentials"
```

It managed to surprise me when seeing the name. 😮